pypit/tests/test_arvcorr.py

```diff
@@ -58,6 +58,7 @@ def test_geocorrect(fitsdict):
     # Specobjs
     specobjs = arspecobj.dummy_specobj(fitsdict, extraction=True)
     slf._specobjs[0] = [specobjs]
+    slf._maskslits[0] = np.array([False]*len(slf._specobjs[0]))
```

Comment:
I think this is slightly quicker:
slf._maskslits[0] = np.zeros(len(slf._specobjs[0]), dtype=np.bool)